### DISCUSSION TITLE
Romanian not supported

### DISCUSSION THREAD

**Discussion by vladxela** [SubtitleEdit/subtitleedit#6574] [General]
Czech, 10 million population, Greek 13 million population supported. Romanian with 20 million population, not supported.

**Comment by niksedk** [comment]
What and where?

Could you explain what is not working for you?
See more here: https://www.nikse.dk/subtitleedit/help#good-github-issue

**Comment by catalin-duta** [comment]
Not sure exactly what the context is, but it is pretty much supported. To not go into too many details:
- The correct* encoding (usually win-1250 or iso-8859-2) is not auto-detected. You need to import the files and choose the encoding manually.
- The really correct encoding would be iso-8859-16, which unfortunately is not available.

**Comment by niksedk** [comment]
@catalin-duta: thx for explaining :)
Also, do let me know if you find any subtitles where the ANSI encoding is not detected automatically - and attach the sub file(s).

@vladxela: Read more here: https://www.nikse.dk/subtitleedit/help#formats and use "windows-1250" or "iso-8859-2" as @catalin-duta  suggested.
Note that UTF8 is recommended for subtitles today :)
